commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/generators/DraftBuilderGenerator.java

```diff
@@ -83,7 +81,11 @@ public TypeSpec generateType(final TypeElement resourceDraftValueType) {
         builder.addFields(fieldSpecs)
                 .addMethod(createDefaultConstructor(constructorModifiers))
                 .addMethod(createConstructor(properties, constructorModifiers))
-                .addMethods(builderMethodSpecs);
+                .addMethods(builderMethodSpecs)
+                .addMethods(createPlusLists(resourceDraftValueType,properties))
+                .addMethods(createPlusTolListElementsMethodSpec(resourceDraftValueType,properties))
+                .addMethods(createPlusSetsMethodSpec(resourceDraftValueType,properties))
+                .addMethods(createPlusElementToSetMethodSpec(resourceDraftValueType,properties));
```

Comment:
I would suggest to use  a more consitent naming scheme here:
```
.addMethods(createPlusListMethods(resourceDraftValueType,properties))
.addMethods(createPlusListElementMethods(resourceDraftValueType,properties))
.addMethods(createPlusSetMethods(resourceDraftValueType,properties))
.addMethods(createPlusSetElementMethods(resourceDraftValueType,properties));
```

And we should use the naming scheme on the called methods too.

Reply:
hmmm yep i know its an ugly naming schema 😢 but i got stuck there 🤔 maybe name it: `createPlusers` as with `createWithers` meaning name the methods as 
* `createListPluser`
* `createListLEmentPluser`...
what do you think?

Reply:
I like your idea: `Pluser` is a `Builder`method for lists 👍